packages/office-addin-manifest/src/validate.ts

```diff
@@ -65,47 +65,51 @@ export async function validateManifest(manifestPath: string): Promise<ManifestVa
     try {
         const validation: ManifestValidation = new ManifestValidation();
 
-        // read the manifest file
-        // const manifest = await readManifestFile(manifestPath);
-        const stream = await createReadStream(manifestPath);
-        let response;
-
-        try {
-            response = await fetch("https://validationgateway.omex.office.net/package/api/check?gates=DisableIconDimensionValidation",
-                {
-                    body: stream,
-                    headers: {
-                        "Content-Type": "application/xml",
-                    },
-                    method: "POST",
-                });
-        } catch (err) {
-            throw new Error(`Unable to contact the manifest validation service.\n${err}`);
-        }
-
-        const text = await response.text();
-        const json = JSON.parse(text.trim());
-
-        if (json) {
-            validation.report = json;
-            validation.status = response.status;
-        }
-
-        if (validation.report) {
-            const result = validation.report.status;
-
-            if (result) {
-                switch (result.toLowerCase()) {
-                    case "accepted":
-                        validation.isValid = true;
-                        break;
+        // read the manifest file to ensure the file path is valid
+        const manifest: ManifestInfo = await readManifestFile(manifestPath);
+
+        if (manifest) {
+            const stream = await createReadStream(manifestPath);
+            let response;
+    
+            try {
+                response = await fetch("https://validationgateway.omex.office.net/package/api/check?gates=DisableIconDimensionValidation",
+                    {
+                        body: stream,
+                        headers: {
+                            "Content-Type": "application/xml",
+                        },
+                        method: "POST",
+                    });
+            } catch (err) {
+                throw new Error(`Unable to contact the manifest validation service.\n${err}`);
+            }
+    
+            const text = await response.text();
+            const json = JSON.parse(text.trim());
+    
+            if (json) {
+                validation.report = json;
+                validation.status = response.status;
+            }
+    
+            if (validation.report) {
+                const result = validation.report.status;
+    
+                if (result) {
+                    switch (result.toLowerCase()) {
+                        case "accepted":
+                            validation.isValid = true;
+                            break;
+                    }
                 }
+            } else {
+                throw new Error("The manifest validation service did not return the expected response.");
```

Comment:
I don't think this else can even happen since readManifestFile() should return the manifest info or throw.

You don't actually need the return value from readManifestFile() at all.

Based on the original code where the line was commented out, you should just be able to uncomment the line and remove the variable assignment.